apps/heft/src/cli/HeftToolsCommandLineParser.ts

```diff
@@ -222,13 +221,15 @@ export class HeftToolsCommandLineParser extends CommandLineParser {
     process.exitCode = 0;
   }
 
-  private _normalizeCwd(): void {
-    const buildFolder: string = this._heftConfiguration.buildFolder;
+  private async _normalizeCwdAsync(): Promise<void> {
+    let buildFolder: string = this._heftConfiguration.buildFolder;
     this._terminal.writeLine(`Project build folder is "${buildFolder}"`);
     const currentCwd: string = process.cwd();
     if (currentCwd !== buildFolder) {
       // Update the CWD to the project's build root. Some tools, like Jest, use process.cwd()
       this._terminal.writeVerboseLine(`CWD is "${currentCwd}". Normalizing to project build folder.`);
+      // The reason why we need to redirect to another directory first is to solve the sensitive casing issues in cloud buiild
+      process.chdir(path.resolve('..', __dirname));
```

Comment:
```suggestion
      // If `process.cwd()` and `buildFolder` differ only by casing on Windows, the chdir operation will not fix the casing, which is the entire purpose of the exercise.
      // As such, chdir to a different directory first. That directory needs to exist, so use the parent of the current directory. This will not work if the current folder is the drive root, but that is a rather exotic case.
      process.chdir(path.dirname(buildFolder));
```

Reply:
This needs to be resolved relative to the project. `__dirname` may be nonsensical if the developer is using Yarn.

Reply:
We use `__dirname` all over the place when resolving schemas and no one has complained about that on Yarn. Changing to `__dirname` may actually be the best option here.